src/main/java/frc/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -1,22 +1,38 @@
 package frc.robot.subsystems.gripper;
 
-import static frc.robot.subsystems.intake.IntakeConstants.*;
-
 import edu.wpi.first.units.*;
 import frc.robot.Constants;
 import lib.webconstants.LoggedTunableNumber;
 
 public class GripperConstants {
-    public static final Measure<Angle> INTAKE_ANGLE = null;
-    public static final Measure<Angle> OUTTAKE_ANGLE = null;
-    public static final double INTAKE_POWER = 0;
-    public static final double OUTTAKE_POWER = 0;
+    public static final Measure<Angle> INTAKE_ANGLE =
+            Units.Rotations.of(0); // TODO: replace with actual value
+    public static final Measure<Angle> OUTTAKE_ANGLE =
+            Units.Rotations.of(0); // TODO: replace with actual value
+    public static final double INTAKE_POWER = 0; // TODO; replace with actual value
+    public static final double OUTTAKE_POWER = 0; // TODO: replace with actual value
+    public static final double AMP_POWER_NORMAL = 0; // TODO: replace with actual value
+    public static final double AMP_POWER_REVERSE = 0; // TODO: replace with actual value
+    public static final double TRAP_POWER = 0; // TODO: replace with actual value
+    public static final double SHOOT_POWER = 0; // TODO: replace with actual value
     public static final Measure<Dimensionless> THRESHOLD = Units.Percent.of(0.02);
+    public static final Measure<Distance> GRIPPER_OUTTAKE_MIN_HEIGHT =
+            Units.Meters.of(0); // TODO: replace with actual value
+    public static final Measure<Distance> GRIPPER_LENGTH = Units.Meters.of(0.534_35);
     public static final Measure<Distance> GRIPPER_POSITION_X = Units.Meters.of(0);
     public static final Measure<Distance> GRIPPER_POSITION_Y = Units.Meters.of(0);
     public static final Measure<Distance> GRIPPER_POSITION_z = Units.Meters.of(0.6461);
     public static final double ANGLE_MOTOR_GEAR_RATIO = 58.5;
 
+    public static final Measure<Angle> WRIST_FOLD_POSITION =
+            Units.Rotations.of(0.5); // TODO: replace with actual value
+    public static final Measure<Angle> WRIST_FORWARD_AMP_POSE =
+            Units.Rotations.of(0); // TODO: replace with actual value;
+    public static final Measure<Angle> WRIST_BACKWARDS_AMP =
+            Units.Rotations.of(0.4); // TODO: replace with actual value;
+    public static final Measure<Angle> WRIST_TRAP_ANGLE = Units.Rotations.of(0.4);
+    public static final Measure<Angle> WRIST_TOLERANCE = Units.Rotations.of(0.05);
```

Comment:
All of these are of the same type, and thus should have similar names -- currently one is a "position", another is a "pose", another is an "angle", another is a "tolerance" (should probably be a percentage), and another one is nothing 😢,
Also, I think it'll be much easier to work with degrees in the constants.

Reply:
c3a7045

Reply:
Nice!
Remember that [`Measure.isNear()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/units/Measure.html#isNear(edu.wpi.first.units.Measure,double)) accepts a percentage, so `WRIST_TOLERANCE` should indeed be dimensionless.

Reply:
394acf9